README.md

```diff
@@ -1,9 +1,47 @@
-# Derailed
+# Derailed API
 
-**This is the Derailed Pre-Alpha API Repository.**
+The most powerful API of the lands.
+Handles nearly every interaction except for
+process-heavy interactions, ultra-frequent
+or high-speed interactions, and our real-time infrastructure
+which is instead based on Elixir 
 
-By all means, use this at your own discretion.
-We cannot currently guarantee a bug-free stable experience
-because of the new and quite funamentally big API.
+## Deploying
 
-***Please* do not make any PRs without telling me or else they won't be accepted.**
+Deploying the Derailed API is quite easy, just fill in the
+required `.env` variables and launch either using
+the `gunicorn` command or just running our development build.
+
+You can also use our docker-compose config,
+although this isn't recommended since it
+could make scaling in the future much more
+of a struggle and increase any pain of running
+our services separately.
```

Comment:
what.

Reply:
Migrating something like Redis, or PostgreSQL from a docker compose environment would be much harder than just running them as separated services in the first place

Reply:
you can still easily scale it up tho